about.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Breana Kambs Official Website</title>
+  <link rel="shortcut icon" type="image/png" href="images/favicon.png">
+</head>
+<body>
+  <hr /> 
```

Comment:
Remove the horizontal rule. It is meant to mark a thematic break which does not exist. Use CSS only for visual presentation.